### PR TITLE
chore: modernize package.json and rollup for vNext

### LINKED ISSUE #5960: chore: update vNext package export configuration

### PR DESCRIPTION
**Pull request by EisenbergEffect**
# Pull Request

## 📖 Description

This PR updates package.json metadata, package.json exports, and improves rollups. It includes:

* all packages
  * updated rollup and plugins
  * added homepage metadata to package.json
  * added directory repo metadata to package.json
* fast-element
  * added a production and development build
  * exposed debug and polyfills through exports
  * added polyfills to the rollup build
  * added a debug rollup build
* fast-foundation
  * added polyfills to the rollup build
  * added a debug rollup build
* fast-router
  *  removed the rollup build
* fast-react-wrapper
  * removed the rollup build

### 🎫 Issues

* Closes #5960 

## 👩‍💻 Reviewer Notes

The builds seem to work across packages but a second pair of eyes on the exports and rollup config is appreciated.

## 📑 Test Plan

Running yarn build and inspecting the outputs.

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

## ⏭ Next Steps

I'll be following up with additional PRs that review the actual code exports, deprecations, etc.

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
Looks like the react-wrapper needs some work too. Taking a look.

File: packages/web-components/fast-element/package.json [hunk 1]
```diff
@@ -20,8 +22,17 @@
   "exports": {
     ".": {
       "types": "./dist/fast-element.d.ts",
-      "default": "./dist/esm/index.js"
-
+      "default": "./dist/esm/index.js",
```
Comment (nicholasrice) [inline packages/web-components/fast-element/package.json:25]: I don't know if this is actually impacting anything, but per https://nodejs.org/api/packages.html#conditional-exports the `default` field should come last.

**Comment by EisenbergEffect** [comment]
Ok, build is now passing. Reviews please 😉

**Comment by EisenbergEffect** [comment]
Open Question: Should I remove the `.js` from the debug and polyfills exports for fast-element or leave as is?

**Pull request merged**